test/features/event-checkin.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+Feature('Check In Event', () => {
```

Comment:
don't think this file has anything to do with prebookings? possibly brought over accidently?